client/components/status/status.test.js

```diff
@@ -1745,32 +1759,43 @@ describe("<Status /> interactions", () => {
         }),
       )
       .mockImplementationOnce(() =>
+        // Fifth call fails with 401
         Promise.reject({
           response: {
             status: 401,
             headers: {},
           },
         }),
       );
+
     props = createTestProps();
     wrapper = shallow(<Status {...props} />, {
       context: {setLoading: jest.fn()},
       disableLifecycleMethods: true,
     });
-    jest.spyOn(wrapper.instance(), "getUserRadiusUsage");
+
+    // Test first call - 500 error
     wrapper.instance().getUserRadiusUsage();
     await tick();
     expect(wrapper.instance().state.radiusUsageSpinner).toBe(true);
+
+    // Test second call - get checks
     wrapper.instance().getUserRadiusUsage();
     await tick();
     expect(wrapper.instance().state.userChecks.length).toBe(1);
+
+    // Test third call - get plan
     wrapper.instance().getUserRadiusUsage();
     await tick();
     expect(wrapper.instance().state.userPlan.is_free).toBe(true);
```

Comment:
In this scenario, when the response didn't include checks, we should verify that `showRadiusUsage` was set to false.